build.gradle

```diff
@@ -179,7 +182,7 @@ dependencies {
     compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
     compile group: 'com.rometools', name: 'rome', version: '1.5.1'
 
-    compile group: 'io.aexp.nodes.graphql', name: 'nodes', version: nodes_version
+    compile group: 'io.github.kobylynskyi', name: 'graphql-java-codegen', version: '3.0.0'
```

Comment:
Miksi GraphQL-kirjasto vaihdettiin?
Uudessa Opintonissa on tuo aexp.

Reply:
otin ton kirjaston mukaan koska se generoi graphql rajapinnassa tarvittavat DTO luokat, varsinaista clienttiä siinä ei ole ja jutustelu sisuun päin tehdään resttemplatella. Mielestäni tämä on kivaa koska ei tule turhaa käsin ylläpidettävää dto koodia ja jos sisun rajapinta laajenee (esim kyselyiden osalta) heijastuu nämä automaattisesti generoituun malliin kun vaan päivittää projektiin graphql scheman.

Reply:
Pitäisikö mainita ihan READMEssa, että pitää ajaa ./gradlew build että IDE ei niin punastelisi.